server/status.go

```diff
@@ -39,68 +38,73 @@ import (
 )
 
 const (
-	// stackTraceApproxSize is the approximate size of a goroutine stack trace.
-	stackTraceApproxSize = 1024
-
-	// statusKeyPrefix is the root of the RESTful cluster statistics and metrics API.
-	statusKeyPrefix = "/_status/"
-
-	// statusGossipKeyPrefix exposes a view of the gossip network.
-	statusGossipKeyPrefix = statusKeyPrefix + "gossip"
-
-	// statusLocalKeyPrefix is the key prefix for all local status
-	// info. Unadorned, the URL exposes the status of the node serving
-	// the request.  This is equivalent to GETing
-	// statusNodesKeyPrefix/<current-node-id>.  Useful for debugging
-	// nodes that aren't communicating with the cluster properly.
-	statusLocalKeyPrefix = statusKeyPrefix + "local/"
-
-	// statusLocalLogKeyPrefix exposes a list of log files for the node.
-	// logfiles -> lists available log files
-	// logfiles/ -> lists available log files
-	// logfiles/{file} -> fetches contents of named log
-	statusLocalLogFileKeyPrefix = statusLocalKeyPrefix + "logfiles/"
-	// statusLocalLogFileKeyPattern is the pattern to match
-	// logfiles/{file}
-	statusLocalLogFileKeyPattern = statusLocalLogFileKeyPrefix + ":file"
-
-	// statusLocalStacksKey exposes stack traces of running goroutines.
-	statusLocalStacksKey = statusLocalKeyPrefix + "stacks"
-
-	// statusNodeKeyPrefix exposes status for each of the nodes the cluster.
-	// nodes -> lists all nodes
-	// nodes/ -> lists all nodes
-	// nodes/{NodeID} -> shows only the status for that specific node
-	statusNodeKeyPrefix = statusKeyPrefix + "nodes/"
-	// statusNodeKeyPattern is the pattern to match nodes/{NodeID}
-	statusNodeKeyPattern = statusNodeKeyPrefix + ":id"
-
-	// statusStoreKeyPrefix exposes status for each store.
-	// stores -> lists all nodes
-	// stores/ -> lists all nodes
-	// stores/{StoreID} -> shows only the status for that specific store
-	statusStoreKeyPrefix = statusKeyPrefix + "stores/"
-	// statusStoreKeyPattern is the pattern to match stores/{StoreID}
-	statusStoreKeyPattern = statusStoreKeyPrefix + ":id"
-
-	// statusLogKeyPrefix exposes the logs for each node
-	// logs -> list log entries for the local node
-	// logs/ -> list log entries for the local node
-	// logs/{NodeID} -> list logs entries for that specific node
-	statusLogKeyPrefix = statusKeyPrefix + "logs/"
-	// statusLogKeyPattern is the pattern to match log/{NodeID}
-	statusLogKeyPattern = statusLogKeyPrefix + ":node_id"
-
-	// statusTransactionsKeyPrefix exposes transaction statistics.
-	statusTransactionsKeyPrefix = statusKeyPrefix + "txns/"
-
+	/*
+		Note that :node_id can always be replaced by the value "local" to see
+		the local nodes response.
+
+		/_status/details/:node_id		 - specific node's details
```

Comment:
I starting coding this up, but it breaks httprouter's prefix tree.
It's not hard to code but I would need to use a sub router.
How strongly do you feel about this?